Source/Common/CrossProcessMutex.h

```diff
@@ -128,7 +128,7 @@ class CrossProcessMutex
 public:
     CrossProcessMutex(const std::string& name)
         : m_fd(-1),
-          m_fileName("/var/lock/" + name)
+          m_fileName("/tmp/" + name)
```

Comment:
> m_fileName("/tmp/" + name) [](start = 9, length = 27)

it is agreed that this issue needs a solution, but this should be solved in an user-configurable way, not through a static assignment